### ISSUE TITLE
1.2.0 package may have been published with 1.1.0 dist code

### ISSUE THREAD

**Issue by Skarbo** [nutboltu/storybook-addon-mock#43]
Hey @nutboltu,

It looks like 1.2.0 dist folder contains 1.1.0 dist code.
Maybe 1.2.0 was published with the old dist folder.

At least `utils/faker~getNormalizedUrl` does not contain the new `baseUrl` support.

Anyways, nice plugin!

**Comment by Skarbo** [comment]
While I'm on it; React devtools gives out a React JSON editor warning:

```
[react-json-editor-ajrm - Deprecation Warning] You did not provide a 'locale' prop for your JSON input - This will be required in a future version. English has been set as a default. 
```

**Comment by nutboltu** [comment]
@Skarbo you are right. I have mistakenly released the same dist for 1.2.0. I have depreciated this version and the new code will be found in `1.3.0` version.

FYI, `react-json-editor-ajrm` warning is also fixed.
